notebooks/argo-file-checker.ipynb

```diff
@@ -0,0 +1,417 @@
+    "import time\n",
+    "from ipywidgets import FileUpload, Button, VBox, Output\n",
+    "from IPython.display import display, HTML\n",
+    "import logging\n",
+    "from pathlib import Path\n",
+    "from typing import Any\n",
+    "logger = logging.getLogger(__name__)\n",
+    "\n",
+    "ch = logging.StreamHandler()\n",
+    "ch.setLevel(logging.INFO)\n",
+    "if not logger.hasHandlers():\n",
+    "    logger.addHandler(ch)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4b260a76",
+   "metadata": {},
+   "source": [
+    "# File Checker Environment Setup\n",
+    "===============================\n",
+    "\n",
+    "1. Set the API_BASE_URL to where the File Checker API is running.\n",
+    " - If you are running the File Checker locally inside Docker, set it to the address where the \n",
+    "   Docker container is exposing the API `http://localhost:8000`\n",
+    " - If you are running the File Checker in a Kubernetes cluster, use the cluster\n",
+    "   URL where the service is exposed. For example:\n",
+    "   `https://livkrakentst.clusters.bodc.me/ewetchy/amrit/argo-toolbox/api/file-checker`\n",
+    " - Only one of the URLs should be uncommented based on your environment.\n",
+    "2. Set the default DAC for validation.\n",
+    "3. Do NOT run other cells until you configure this."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 95,
+   "id": "5820cce1",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "🔗 API Base URL:http://localhost:8000\n",
+      "🏛️ DAC: bodc\n"
+     ]
+    }
+   ],
+   "source": [
+    "\n",
+    "# ===============================\n",
+    "# 2️⃣ Configuration\n",
+    "# ===============================\n",
+    "# Local Docker instance\n",
+    "API_BASE_URL=\"http://localhost:8000\"\n",
+    "\n",
+    "# Kubernetes test instance\n",
+    "# API_BASE_URL= \"https://livkrakentst.clusters.bodc.me/ewetchy/amrit/argo-toolbox/api/file-checker\"\n",
+    "\n",
+    "# Default DAC for validation\n",
+    "DEFAULT_DAC = \"bodc\"\n",
+    "\n",
+    "# Endpoints\n",
+    "CHECK_FILE_ENDPOINT = \"/check-files\"\n",
+    "HEALTH_ENDPOINT = \"/\"\n",
+    "\n",
+    "# Request settings\n",
+    "TIMEOUT = 30  # API request timeout in seconds\n",
+    "HEADERS = {\n",
+    "    \"accept\": \"application/json\"\n",
+    "}\n",
+    "\n",
+    "# Full URL for file check endpoint\n",
+    "FILE_CHECK_URL = f\"{API_BASE_URL}/{CHECK_FILE_ENDPOINT}\"\n",
+    "\n",
+    "logger.info(\"🔗 API Base URL:%s\",  API_BASE_URL)\n",
+    "logger.info(\"🏛️ DAC: %s\" ,DEFAULT_DAC)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 102,
+   "id": "48a5fa1a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# ============================================\n",
+    "#  3️⃣ function to test config and api\n",
+    "# ============================================\n",
+    "def test_api_connection() -> bool:\n",
```

Comment:
The functions are overkill, I think. I wouldn't expect a PI to be writing functions in their notebooks.